### PR TITLE
Makefile update.

### PR DESCRIPTION
**Pull request by awdavies**
Fixes issue #19: moving the link statements to the end of the flag
declarations appears to fix the build issue.

### CODE REVIEW

**Comment by awdavies** [comment]
Done.  PTAL

File: linux/example/Makefile [hunk 1]
```diff
@@ -75,7 +76,7 @@ $(FLUTTER_EXAMPLE_DIR)/build:
 		--local-engine=$(FLUTTER_ENGINE_BUILD);
 
 $(BIN_OUT): $(SOURCES) $(LIBRARIES)
-	$(CXX) $(SOURCES) $(CXXFLAGS) -o $@
+	$(CXX) $(SOURCES) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@
```
Comment (stuartmorgan) [inline linux/example/Makefile]: Let's keep C*FLAGS before the sources list.

Comment (awdavies) [reply linux/example/Makefile]: Done

**Comment by awdavies** [comment]
Closes #19

**Pull request merged**